logging_format/whitelist.py

```diff
@@ -0,0 +1,34 @@
+"""
+A logging extra keyword argument whitelist.
+
+"""
+from pkg_resources import iter_entry_points
+
+
+class Whitelist(object):
+    """
+    A pluggable whitelist.
+
+    Uses entry points.
+
+    """
+    def __init__(self, group="logging.extra.whitelist"):
```

Comment:
I chose to use entry points to define the whitelist. My vision is that `marquez` will define *our* whitelist and both `microcosm-logging` and this project will use the same whitelist.

I also allow the whitelist group to be changed for testing purposes.